script/linux_share_build.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+
+
+function install_prerequisites() {
+    TZ=America/Chicago
+    ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+    apt-get update -y
+    apt-get install -y g++ libeigen3-dev libhdf5-dev patchelf git cmake
+}
+
+function setup_folder() {
+    cd ${CURRENT}
+    mkdir ${PLUGIN_DIR}
+    echo "Building the Trelis plugin in ${CURRENT}\\${PLUGIN_DIR}"
+
+    unset LD_LIBRARY_PATH
```

Comment:
if you run this on your computer one want to be sure that the LD_LIBRARY_PATH is not set to ovoid conflict...

Reply:
@pshriwise I believe this is the only outdenting comment.

BU this was done on purpose for safety....

Reply:
Gotcha. I was understanding these as intended for use in docker only, meaning we'd have a fresh environment. Nice to be able to use them in the native kernel though -- have we tested that? 

All that aside, would this command be more appropriate in the `build_plugin.sh` file? It would be more clear that the `LD_LIBRARY_PATH` is unset by these scripts I think.